test/basic.html

```diff
@@ -46,31 +46,29 @@
 
       test('check button via click', function() {
         r1.addEventListener('click', function() {
-          assert.isTrue(r1.getAttribute('aria-checked'));
+          assert.isTrue(r1.getAttribute('aria-checked') == 'true');
           assert.isTrue(r1.checked);
-          done();
         });
-        MockInteractions.down(r1);
+        MockInteractions.tap(r1);
       });
 
       test('toggle button via click', function() {
         r1.checked = true;
         r1.addEventListener('click', function() {
-          assert.isFalse(r1.getAttribute('aria-checked'));
+          assert.isFalse(r1.getAttribute('aria-checked') != 'false');
           assert.isFalse(r1.checked);
-          done();
```

Comment:
This is incorrect, and I think it accidentally got broken and never fixed :(. You should keep the `done();` statement here, and add it as a parameter to the test (i.e. in L47), etc.